test/DynamoCoreTests/NodeExecutionTest.cs

```diff
@@ -654,6 +654,8 @@ public void LoadGraphWithFreezeNodes_Defect9274()
             //Check whether the geometry node is frozen
             var node = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace("8163332d-21ec-4257-9a5a-0b69462db44f");
             Assert.IsTrue(node.IsFrozen);
+            //Frozen nodes should not get involved in execution.
+            Assert.IsFalse(node.WasInvolvedInExecution);
```

Comment:
simple test to check whether the frozen node when opening the graph was involved in execution